js/main.js

```diff
@@ -375,21 +384,26 @@ function addItemsToGearBag( items ) {
 	var DBOpenRequest = window.indexedDB.open("gearBag", 1);
 	DBOpenRequest.onsuccess = function( event ) {
 		db = DBOpenRequest.result;
-		var transaction = db.transaction(["items"], "readwrite");
+		var transaction = db.transaction("items", "readwrite");
 		transaction.onerror = function( event ) {
 			console.log( 'error on transaction', event );
 		}
 		transaction.oncomplete = function( event ) {
 		}
 		
+		/*
+			the For in loop was removed because it was adding a function to the object
+			that the loop was retrieving from the array. The object would be cloned and placed
+			in the indexedDB but then the function would fail because the engine doesn't know 
+			how to clone a function. and since there should only be one item being	
+			dragged and dropped at a time, adding arrays of items is pointless work.
+		*/
```

Comment:
This comment would make more sense as a commit description. As it is, it's referencing code that is no longer here.